orunmila.go

```diff
@@ -268,16 +270,11 @@ func createDbFileifNotExists(dbPtr string) {
 	}
 }
 
-// parse args of the import subcommand and exec it
-func vacuumSubcmd(args []string) {
-	flag := flag.NewFlagSet("vacuum", flag.ContinueOnError)
-
-	flag.Parse(args)
-
-	createDbFileifNotExists(*dbPtr)
+// parse args of the vaccum subcommand and exec it
+func vacuumSubcmd() {
+	// parse the global args instead (debug && db path)
```

Comment:
How about we remove the comment and add a usage for this baby ?

Reply:
on it's way